source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -225,6 +226,15 @@ JsonTranscoderConfig::JsonTranscoderConfig(
   match_incoming_request_route_ = proto_config.match_incoming_request_route();
   ignore_unknown_query_parameters_ = proto_config.ignore_unknown_query_parameters();
   request_validation_options_ = proto_config.request_validation_options();
+  if (proto_config.has_grpc_status_proto_transformer()) {
+    auto* factory = Envoy::Config::Utility::getFactoryByName<
```

Comment:
Any reason not to use `getAndCheckFactoryByName`?
This way if there's an invalid extension name the config will be rejected.

Reply:
Done.